src/OpenLoco/src/Objects/LevelCrossingObject.h

```diff
@@ -27,7 +27,8 @@ namespace OpenLoco
         uint8_t animationSpeed; // 0x07
         uint8_t closingFrames;  // 0x08
         uint8_t closedFrames;   // 0x09
-        uint8_t pad_0A[0x0C - 0x0A];
+        uint8_t var_0A;         // 0x0A
```

Comment:
my favourite question - do you have any ideas for names/what these variables do? also `unk2` in the  enum below

Reply:
also is there a var_0B or is that padding? or is 0A uint16?